.eslintrc.json

```diff
@@ -3,10 +3,25 @@
     "jasmine": true
   },
   "extends": "@wireapp/eslint-config",
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {
+        "@typescript-eslint/member-ordering": "off",
+        "@typescript-eslint/typedef": "off",
+        "no-unused-vars": "off"
+      }
+    }
+  ],
+  "parserOptions": {
+    "project": ["./tsconfig.json", "./tsconfig.bin.json", "./tsconfig.mocha.json"]
+  },
   "plugins": ["jasmine"],
   "rules": {
-    "handle-callback-err": 2,
-    "import/no-default-export": 0,
-    "no-undef": 0
+    "import/no-default-export": "off",
```

Comment:
This needs to stay since it makes more sense to export React components as default.

Reply:
Shall we remove that form the package then for all projects?! Most of the project will have some kind of React in it..

Reply:
Mh no, I would like to find out first if we can somehow tell ESLint to disable some rules for certain directories...